UI/cmake/feature-youtube.cmake

```diff
@@ -0,0 +1,14 @@
+if(YOUTUBE_CLIENTID
+   AND YOUTUBE_SECRET
+   AND YOUTUBE_CLIENTID_HASH
+   AND YOUTUBE_SECRET_HASH
+   AND TARGET OBS::browser-panels)
```

Comment:
YouTube integration does not require obs-browser. If browser is disabled, the chat dock is not added.